### PR TITLE
Migrate "Extra" class commands to Commander 2

### PR DESCRIPTION
**Pull request by hernanmd**
IMPORTANT: This PR should be applied after #17099

This PR migrates the "Extra" class commands from the contextual menu (while selecting a class or multiple classes) from Calypso-Browser to SystemCommands-ClassCommands, as described in #17100

### CODE REVIEW

**Comment by jecisc** [comment]
@hernanmd Can you merge Pharo13 into this branch since I merged your other PR?

File: src/BaselineOfCalypso/BaselineOfCalypso.class.st [hunk 1]
```diff
@@ -98,9 +98,6 @@ BaselineOfCalypso >> baseline: spec [
 			package: #'Calypso-SystemPlugins-Deprecation-Queries-Tests'
 			with: [
 				spec requires: #( #'Calypso-SystemPlugins-Deprecation-Queries' ) ];
-			package: #'Calypso-SystemPlugins-FileOut-Browser' with: [
```
Comment (jecisc) [inline src/BaselineOfCalypso/BaselineOfCalypso.class.st:101]: I see this package removed from the baseline but the code is not removed from the code base of Pharo. This package should be removed.

**Review by jecisc — CHANGES_REQUESTED** [review]

File: src/BaselineOfCalypso/BaselineOfCalypso.class.st (hunk 1 above)
Comment (hernanmd) [reply src/BaselineOfCalypso/BaselineOfCalypso.class.st:101]: Done in https://github.com/pharo-project/pharo/pull/17101/commits/a4cd1a5da020b2870d879db4d5a327020334f346

**Comment by hernanmd** [comment]
Failure is not related: unix-64 / Tests-unix-64 / Unix64.UnifiedFFI.Tests.LibTTYTest.test2

**Review by jecisc — APPROVED** [review]

**Pull request merged**